src/resources/BreadCrumbResource.ts

```diff
@@ -66,3 +66,11 @@ export function getStaffDashboardBreadcrumb (): BreadcrumbIF {
     href: `${sessionStorage.getItem('BUSINESSES_URL')}staff/${getParams()}`
   }
 }
+
+export function getStaffRegistryBreadcrumb (): BreadcrumbIF {
+  const accountId = sessionStorage.getItem('ACCOUNT_ID')
```

Comment:
This should be handled by `navigate()` instead.

Reply:
Oh wait, this is different.

Looks fine 👍

Reply:
Actually, the ideal solution might be for Auth Web to redirect from, say:
`https://dev.bcregistry.ca/business/auth/account/business?accountid=2288`
to:
`https://dev.bcregistry.ca/business/auth/account/668/business`

(It partially does this already for non-staff/IDIR logins.)

Do you want to try looking into this?

Reply:
Yes. I need to try the Auth Web. Because it didn't redirect properly with my way.

Reply:
The reason I recommend a fix to Auth Web is because many UIs/web apps have this same breadcrumb feature, and all would need to be updated now and any time the URL changes. (The URL is otherwise provided by external config, so no rebuilding needed.)

If the problem is that Auth Web isn't going to the right business page when the user is staff/IDIR, then that should be fixed directly.

OTOH, if the PO decides to accept this temporary fix now, and we have a ticket to implement a proper fix and delete this workaround later, then I'm OK with this.

Reply:
This is a good question. Yui and Tracy did suggest to work on this fix in another ticket. I can now see how this has an impact on other UIs, such as filing-ui that doesn't have "My Business Registry" either (on staff). Meanwhile, I am not sure what I need to do with Auth Web. If "My Business Registry" should appear in all UIs, then it's better to work on it in another ticket and discard any change in this ticket.

Reply:
What's the other ticket? Perhaps this needs a larger discussion... please raise it at Standup. Thx.

If we take the breadcrumbs out of this ticket, does THIS ticket need to change any code?

Reply:
If we take the breadcrumbs out of this ticket, then this ticket won't have any change.

Reply:
When I am logged in as staff (IDIR), the following URL doesn't redirect to My Business Registry. Do I have the right URL?
https://dev.bcregistry.ca/business/auth/account/668/business

Reply:
Ohhhhh, I think I know!

If staff start from the [staff dashboard[(https://dev.bcregistry.ca/business/auth/staff/dashboard/active) and go the Entity Dashboard or Manage Account pages then a bunch of account-related fields are set in session storage and those pages display correctly.

However, if staff directly go to a business' dashboard (eg, https://dev.bcregistry.ca/business/FM1000007) then those session variables are not set the and going to My Business Registry (eg, https://dev.bcregistry.ca/business/auth/account/668/business) doesn't work.

All this to say this is more complicated than initially believed.

And also this change has been removed from this ticket.

Reply:
Yes, you are right. "My Business Registry" on the breadcrumbs can redirect to either of the two pages: staff's registry page or the user's registry page. Suppose we have it in the breadcrumbs for both cases. 
1. If a staff is accessing a user's NR and wants to return to the user's registry page, then this button should do that. 
2. If a staff is accessing his/her own NR and wants to return his/her own registry page, then this button also needs to do that. 
This button is very complicated and I feel sorry to bring it up. 

Since BN isn't required for SP, there isn't any change at all in this branch. 

I think it's safe to close this pull request.

Reply:
Agreed. I'm resolving this conversation.